tests/tests_ptp_multi.yml

```diff
@@ -9,6 +9,7 @@
         interfaces: "{{ [ ansible_default_ipv4['interface'] ] }}"
         delay: 0.001
         transport: UDPv4
+        hybrid_e2e: yes
```

Comment:
how does the test verify that this setting is in effect?

Reply:
It doesn't. It just checks if ptp4l starts, same as with the other options.